src/Symfony/Component/Form/Form.php

```diff
@@ -1112,7 +1112,7 @@ private function viewToNorm($value)
                 $value = $transformers[$i]->reverseTransform($value);
             }
         } catch (TransformationFailedException $exception) {
-            throw new TransformationFailedException(sprintf('Unable to reverse value for property path "%s": '.$exception->getMessage(), $this->getPropertyPath()), $exception->getCode(), $exception);
+            throw new TransformationFailedException(sprintf('Unable to reverse value for property path "%s": "%s".', $this->getPropertyPath(), $exception->getMessage()), $exception->getCode(), $exception);
```

Comment:
in order to remove the extra double quotes, the concatenation should be moved outside the sprintf

Reply:
fixed that. i see 3 more cases like this in the same file, but i don't know how i would reproduce the error. should i put the same fix?

Reply:
yes please.

Reply:
btw, we have this issue in other places too. This has been introduced in https://github.com/symfony/symfony/pull/36437

Reply:
@stof, i also fixed the files you pointed out there.